### DISCUSSION TITLE
Per mp3 offset instead of per beatmap offset.

### DISCUSSION THREAD

**Discussion by rikimasan** [ppy/osu#25299] [Ideas] [answered]
After a somewhat recent update to lazer, offsets for beatmaps are handled per beatmap. This works great in the majority of situations since most beatmaps (and all ranked ones to my knowledge) use 1 mp3 for the whole mapset. However, it is very common in practice mapsets to have multiple mp3 files such as with [this beatmap set](https://osu.ppy.sh/beatmapsets/72474#osu/207692).  These beatmaps are typically graveyarded. I would argue that since ranked beatmaps should have correct offsets to be ranked it is actually more important to cater towards these graveyarded beatmaps that would typically have offset issues when thinking about local offset.

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/issues/24558
